lib/SPIRV/libSPIRV/SPIRVEnum.h

```diff
@@ -237,6 +237,7 @@ template <> inline void SPIRVMap<SPIRVExecutionModeKind, SPIRVCapVec>::init() {
   ADD_VEC_INIT(ExecutionModeOutputTriangleStrip, {CapabilityGeometry});
   ADD_VEC_INIT(ExecutionModeVecTypeHint, {CapabilityKernel});
   ADD_VEC_INIT(ExecutionModeContractionOff, {CapabilityKernel});
+  ADD_VEC_INIT(ExecutionModeSubgroupSize, {CapabilitySubgroupDispatch});
```

Comment:
Could you please update corresponding test with check for new capability?

https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/3bc9ff061d61558394ffed27a6ee0a83f75d2dcd/test/transcoding/ReqdSubgroupSize.ll#L10-L11

Reply:
Forgot about this
Fixed